server/resource_server/controller/controller.go

```diff
@@ -11,7 +11,7 @@ import (
 
 	"github.com/golang-jwt/jwt/v4"
 
-	"globe-and-citizen/layer8/proxy/resource_server/config"
+	"globe-and-citizen/layer8/proxy/config"
```

Comment:
I wonder if we should rename the 'go.mod' of 'layer8/server'. Currently it is called `globe-and-citizen/layer8/proxy` and I think this is a legacy detail.

Reply:
Yes, I am changing it to `globe-and-citizen/layer8` , and since we have the resource server, oauth and the proxy all in one, I think this general name is better or we can use `globe-and-citizen/layer8/server` also.